data/mods/Aftershock/recipes/uncraft.json

```diff
@@ -107,11 +107,335 @@
     ]
   },
   {
-    "result": "afs_atomic_smartphone",
+    "result": "aza_sword",
     "type": "uncraft",
     "activity_level": "fake",
     "time": 500,
     "qualities": [ { "id": "HAMMER", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
     "components": [ [ [ "afs_material_2", 6 ] ], [ [ "afs_material_3", 4 ] ], [ [ "afs_material_1", 8 ] ] ]
+  },
+  {
+    "result": "broken_utilibot",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "2 h",
+    "skill_used": "electronics",
+    "difficulty": 4,
+    "using": [ [ "soldering_standard", 20 ], [ "welding_standard", 5 ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
+    "components": [
+      [ [ "ai_module", 1 ] ],
+      [ [ "robot_controls", 1 ] ],
+      [ [ "sensor_module", 1 ] ],
+      [ [ "utilibot_chassis", 1 ] ],
+      [ [ "afs_circuitry_1", 6 ] ],
+      [ [ "afs_circuitry_2", 4 ] ],
+      [ [ "afs_circuitry_3", 2 ] ],
+      [ [ "afs_energy_storage_3", 2 ] ],
+      [ [ "afs_energy_storage_4", 1 ] ],
+      [ [ "afs_material_1", 12 ] ],
+      [ [ "afs_material_2", 4 ] ]
+    ]
+  },
+  {
+    "result": "broken_hazmatbot",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "2 h",
+    "skill_used": "electronics",
+    "difficulty": 4,
+    "using": [ [ "soldering_standard", 20 ], [ "welding_standard", 5 ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
+    "components": [
+      [ [ "ai_module", 1 ] ],
+      [ [ "robot_controls", 1 ] ],
+      [ [ "sensor_module", 1 ] ],
+      [ [ "hazmatbot_chassis", 1 ] ],
+      [ [ "afs_circuitry_1", 6 ] ],
+      [ [ "afs_circuitry_2", 4 ] ],
+      [ [ "afs_circuitry_3", 2 ] ],
+      [ [ "afs_energy_storage_3", 2 ] ],
+      [ [ "afs_energy_storage_4", 1 ] ],
+      [ [ "afs_material_1", 12 ] ],
+      [ [ "afs_material_2", 4 ] ]
+    ]
+  },
+  {
+    "result": "broken_utilibot_butler",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "2 h",
+    "skill_used": "electronics",
+    "difficulty": 5,
+    "using": [ [ "soldering_standard", 20 ], [ "welding_standard", 5 ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
+    "components": [
+      [ [ "ai_module", 1 ] ],
+      [ [ "robot_controls", 1 ] ],
+      [ [ "sensor_module", 1 ] ],
+      [ [ "hazmatbot_chassis", 1 ] ],
+      [ [ "afs_circuitry_1", 6 ] ],
+      [ [ "afs_circuitry_2", 4 ] ],
+      [ [ "afs_circuitry_3", 2 ] ],
```

Comment:
The circuitry substitutes for RAM? This one should probably have more of the advanced circuitry.

Reply:
Yes, circuitry is substituted for ram.